bindings/python/pymongoarrow/types.py

```diff
@@ -64,8 +71,13 @@ def _get_internal_typemap(typemap):
     internal_typemap = {}
     for fname, ftype in typemap.items():
         for checker, internal_id in _TYPE_CHECKER_TO_INTERNAL_TYPE.items():
-            if checker(ftype):
-                internal_typemap[fname] = internal_id
-                continue
+            # Catch error where the pyarrow checkers are looking for an `id`
+            # attribute that might not exist on non-pyarrow types
+            # (like ObjectId).
```

Comment:
I don't follow this comment. Can you give an example?

Reply:
Updated